### PR TITLE
Make docs more useful

### PR DESCRIPTION
**Pull request by feanil**
Get the docs to be a bit more useful in preparation for
shipping them to rtd.

### CODE REVIEW

**Comment by feanil** [comment]
Current dependencies on/for this PR:
* master
  * **PR #50** <a href="https://app.graphite.dev/github/pr/openedx/DoneXBlock/50" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/openedx/DoneXBlock/50?utm_source=stack-comment).

File: docs/source/concepts/index.rst [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-.. toctree::
-   :glob:
+When learners are going through a learning content, it's sometimes useful to
+provide interactive check points.  The DoneXBlock gives the learner a simple way
+to indicate that they have completed some piece of work.
 
-   *
+It's especially useful if the work is off-platform in some other site, or
+physical work that the need to do.
```
Comment (e0d) [inline docs/source/concepts/index.rst]: typo: that **they**

**Review by e0d — COMMENTED** [review]
Committed a one letter typo fix.  Need one clarification, but otherwise looks good.

File: docs/source/decisions/0000-purpose-of-this-repo.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Accepted
+
+Context
+*******
+
+We need a way to track when learners home completed content that can't be easily
+track by some other mechanism or where we want the learner to conciously mark
+that they have finished some piece of work, even if say the work is only
+partially done.
```
[the remark this answers is not in the document]
Comment (feanil) [reply docs/source/decisions/0000-purpose-of-this-repo.rst]: This was a bad example, I was trying to say, you could mark a thing as done even if you weren't actually done with it but that's not a very helpful example so changed it up.

**Review by e0d — APPROVED** [review]

**Pull request merged**